lms/djangoapps/teams/views.py

```diff
@@ -619,6 +623,9 @@ class TeamsDetailView(ExpandableFieldViewMixin, RetrievePatchAPIView):
                 * last_activity_at: The date of the last activity of any team member
                   within the team.
 
+                * organization_protected: Whether the team consist of team members in
```

Comment:
nit: `Whether the team consists of organization-protected learners` may reads a bit better